.github/workflows/lint.yml

```diff
@@ -0,0 +1,79 @@
+name: Lint
+
+on:
+  pull_request:
```

Comment:
This needs to be run in many more places, at least in the cron tests, on merge, and then `/run-acceptance-tests` is called.